packages/jsActions/nanoflow-actions-native/src/geolocation/GetCurrentLocation.ts

```diff
@@ -52,14 +68,20 @@ export async function GetCurrentLocation(
             });
         }
 
-        function onError(error: GeolocationError): void {
+        function onError(error: GeolocationError | GeoError): void {
             return reject(new Error(error.message));
         }
 
-        function getOptions(): GeolocationOptions {
-            const timeoutNumber = timeout && Number(timeout.toString());
+        function getOptions(): GeolocationOptions | GeoOptions {
+            let timeoutNumber = timeout && Number(timeout.toString());
             const maximumAgeNumber = maximumAge && Number(maximumAge.toString());
 
+            if (timeoutNumber === undefined && Platform.OS === "ios") {
+                timeoutNumber = 30000;
+            } else if (timeoutNumber === 0) {
```

Comment:
Do you need to change the timeout for both Android and iOS when the timeout is 0? I thought when we spoke only iOS crashes when timeout is 0?